### ISSUE TITLE
Visisted GK trackable shown in cache inventory

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#15263]
### Describe your problem!

I guess this could be more a server side problem on GeoKrety but let me still open an issue here to document it and find out:
I do visit a certain GeoKret to all caches I find (via the c:geo visiting function while logging).
Now I noticed, that this GK is showing to be in this caches inventory where it has last visited.

@kumy Can you take a look into this strange issue, please.

### How to reproduce?

- Open GC2BG94 (GK service needs to be active)
- You see I have found it on 28.01.24
- Go to the inventory tab 

### Actual result after these steps?

It shows "Mein Kilometerzähler" (GKC240) to be located in the cache.
If you open that GK you will however see:
- The "Last seen" is "At Lineflyer" (which is correct)
- The trackables logbook shows in the most recent entry, that this GK only visited the cache

### Expected result after these steps?

The GK should not be shown in the caches inventory

### Reproducible

Yes

### c:geo Version

2024.01.28-NB

**Comment by kumy** [comment]
That make me think about this [issue](https://github.com/geokrety/geokrety-website/issues/867). I have no idea if they can be related or not.

I will try to have a look "soon"

**Comment by kumy** [comment]
@Lineflyer I believe it has been fixed with [GK v2.46.1](https://github.com/geokrety/geokrety-website/releases/tag/v2.46.1)
Would you please confirm :pray: 

---
The concerned endpoint is `/api/v1/export2`, called with parameter `wpt=OC1312E`

### Before v2.46.1
```
$ curl  -qSs "https://geokrety.org/api/v1/export2?wpt=OC1312E" -v|xmlstarlet fo
<?xml version="1.0" encoding="UTF-8"?>
<gkxml version="1.0" date="2024-02-18 11:37:27" date_Iso8601="2024-02-18T11:37:27+00:00">
  <geokrety>
    <geokret id="49728" type="2" owner_id="17135" ownername="Lineflyer" holder_id="17135" holdername="" dist="369602" date="2024-02-10" state="5" last_pos_id="5155123" last_log_id="5155123" places="5037" image="GKC240_6503bf540cf54"><![CDATA[Mein Kilometerzähler]]></geokret>
  </geokrety>
</gkxml>
```

### Starting from v2.46.1
```
$ curl -qSs "https://geokrety.org/api/v1/export2?wpt=OC1312E"|xmlstarlet fo
<?xml version="1.0" encoding="UTF-8"?>
<gkxml version="1.0" date="2024-02-18 11:40:17" date_Iso8601="2024-02-18T11:40:17+00:00">
  <geokrety/>
</gkxml>
```

**Comment by Lineflyer** [comment]
@kumy 
I can confirm, that my GK is no longer listed to be in cache GC2BG94. Not sure though if this is still the relevant testing as the GK has visited more caches since that time.
I also checked the most recent cache I logged this GK as visited and its not in its inventory.

Seems OK!
